clusterloader2/pkg/prometheus/manifests/prometheus-prometheus.yaml

```diff
@@ -13,18 +13,26 @@ spec:
     - name: alertmanager-main
       namespace: monitoring
       port: web
+  logLevel: debug
```

Comment:
It seems that there is an idea to revert this logLevel from original PR - https://github.com/kubernetes/perf-tests/commit/c4c88273493d802a9ec22cdbde0efe8f4308b531#commitcomment-46472916

Could you remove this logLevel line?

Reply:
Done.